CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+## [0.0.43](https://github.com/botpress/studio/compare/v0.0.42...v0.0.43) (2021-12-02)
```

Comment:
I should display the changelog for 0.0.44 not 0.0.43

Reply:
Let me rerun this